src/support/variables/typography.scss

```diff
@@ -32,7 +32,29 @@ $lh-condensed: 1.25 !default;
 $lh-default: 1.5 !default;
 
 // Font stacks
-$body-font: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji" !default;
+// stylelint-disable no-eol-whitespace
+$body-font:
+  // Apple OSs
+  -apple-system,
+  BlinkMacSystemFont,
+  
+  // Windows
+  // Note this should be prioritized over `system-ui` to avoid legacy fonts.
+  // See https://infinnie.github.io/blog/2017/systemui.html
```

Comment:
This post seems to be from 2017? So the browser/OS bugs might be fixed by now? On the other hand, I'm not sure how to test the problems. So maybe still good to keep them. 🤷